cmd/fakemachine/main.go

```diff
@@ -11,6 +11,8 @@ import (
 	"strings"
 )
 
+var Version string = "none"
```

Comment:
Does it make sense to expose this in the fakemachine go API (so we could e.g print the version of fakemachine used in debos debug) ? It could come later after this PR.

Reply:
That's very hard with golang; remember is a static library. Fakemachine, the library, itself has no idea what version it is inherently. Debos could figure it our from the golang buildinfo, but that gets cleared when doing debian builds for some reason

Reply:
Sounds reasonable enough.